example(content): add custom content example (#21)
sahabpardaz/ctablex: examples/content/README.md

# Custom Content example

> In this example, custom content rendering added to columns.

Reviewer: ```suggestion
This example shows how different columns can customize their cells content
```
